lms/static/js/spec/discovery/discovery_spec.js

```diff
@@ -5,16 +5,18 @@ define([
     'common/js/spec_helpers/ajax_helpers',
     'common/js/spec_helpers/template_helpers',
     'js/discovery/discovery_factory',
-    'js/discovery/collection',
-    'js/discovery/form',
-    'js/discovery/result',
-    'js/discovery/result_item_view',
-    'js/discovery/result_list_view',
-    'js/discovery/filter',
-    'js/discovery/filters',
-    'js/discovery/filter_bar_view',
-    'js/discovery/filter_view',
-    'js/discovery/search_facets_view'
+    'js/discovery/views/search_form',
+    'js/discovery/models/course_card',
```

Comment:
Nit: it seems odd to have a single spec file that requires everything (as opposed to individual test files concentrated on particular source files).